deploy/acm/crds/policy.open-cluster-management.io_placementbindings_crd.yaml

```diff
@@ -23,6 +27,17 @@ spec:
               of an object. Servers should convert recognized schemas to the latest
               internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
             type: string
+          bindingOverrides:
+            description: BindingOverrides defines the overrides to the Subjects
+            properties:
+              remediationAction:
+                description: This field overrides the policy remediationAction on
+                  target clusters
+                enum:
+                - Enforce
+                - enforce
```

Comment:
Is this enum really supposed to be "Enforce, enforce" or should it be "inform, enforce"? :thinking:

Reply:
Yeah, it is "Enforce, enforce".  For now, only overriding to enforce is supported.